cmd/podman/containers/create.go

```diff
@@ -137,6 +137,7 @@ func create(cmd *cobra.Command, args []string) error {
 		}
 		cliVals.InitContainerType = initctr
 	}
+	// TODO: v5.0 block users from setting restart policy for a container if the container is in a pod
```

Comment:
I am still strongly objecting to that but we can wrestle that out when time is due :^)